test/unit/native_deps/native_action_inputs_test.bzl

```diff
@@ -0,0 +1,152 @@
+"""Unittests for rust rules."""
+
+load("@bazel_skylib//lib:unittest.bzl", "analysistest", "asserts")
+load(
+    "//rust:defs.bzl",
+    "rust_binary",
+    "rust_library",
+    "rust_proc_macro",
+    "rust_shared_library",
+    "rust_static_library",
+)
+
+def _native_action_inputs_present_test_impl(ctx):
+    env = analysistest.begin(ctx)
+    tut = analysistest.target_under_test(env)
+    action = tut.actions[0]
```

Comment:
Nit: use assert_action_mnemonic from //test/unit:common.bzl to double check we are looking into the right action.

Reply:
Done.